app/CocoaPods/CPPodfileConsoleViewController.swift

```diff
@@ -9,9 +10,57 @@ class CPPodfileConsoleViewController: NSViewController, NSTextViewDelegate {
     super.viewDidLoad()
 
     let settings = CPFontAndColourGateKeeper()
-    textView.font = settings.defaultFont;
+    textView.font = settings.defaultFont
   }
-
+  
+  override func viewWillAppear() {
+    super.viewWillAppear()
+    
+    guard let podfileVC = podfileViewController, project = podfileVC.userProject else {
+      return print("CPPodfileEditorViewController is not set up with a PodfileVC in the VC heirarchy.")
+    }
+    
+    if textView.string!.isEmpty {
+      hintButton.hidden = false
+      if project.contents.isEmpty {
```

Comment:
perhaps this would be better suited to being an `ARPodfileErrorState` enum, then the logic doesn't have to be duplicated below?